.prettierrc.yml

```diff
@@ -0,0 +1,4 @@
+quoteProps: consistent
```

Comment:
:eyes: Did we ever remove this rule in the ESLint config? I had thought there was a general dislike of this on the team.

Reply:
So far, no rejoinders have followed this: https://github.com/MetaMask/eslint-config/issues/175#issuecomment-828788208

Reply:
We're doing `as-needed`.